subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperLoggingIntegrationTest.groovy

```diff
@@ -17,18 +17,26 @@
 package org.gradle.integtests
 
 class WrapperLoggingIntegrationTest extends AbstractWrapperIntegrationSpec {
```

Comment:
:x: We'll need a test for this behavior:

> This change removes the wrapper logs for unzipping and setting
permissions, changing them to only log when something went wrong.